packages/react-examples/src/react/Image/Image.MaximizeFrame.Example.tsx

```diff
@@ -6,7 +6,7 @@ import { Image, IImageProps, ImageFit } from '@fluentui/react/lib/Image';
 const imageProps: IImageProps = {
   maximizeFrame: true,
   imageFit: ImageFit.cover,
-  src: 'http://via.placeholder.com/500x500',
+  src: 'https://via.placeholder.com/600x600',
```

Comment:
The `500x500` image asset doesn't seem to exist anymore on the site so just replaced it with `600x600`